src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -211,14 +210,12 @@ protected IEnumerable<CompletionItem> GetTopLevelItems(ISymbol symbol, TSyntax s
                 items.AddRange(GetParameterItems(symbol.GetParameters(), syntax, ParameterElementName));
                 items.AddRange(GetParameterItems(symbol.GetTypeParameters(), syntax, TypeParameterElementName));
 
-                var property = symbol as IPropertySymbol;
-                if (property != null && !existingTopLevelTags.Contains(ValueElementName))
+                if (symbol is IPropertySymbol property && !existingTopLevelTags.Contains(ValueElementName))
```

Comment:
>property [](start = 46, length = 8)

is this variable used?

Reply:
nope.  addressed.  our fixer could do better here in these cases.